xbundle/__init__.py

```diff
@@ -250,7 +250,12 @@ def import_metadata_from_directory(self, dir):
             policies = etree.Element('policies')
             policies.set('semester',os.path.basename(pdir))
             for fn in glob.glob(join(pdir, '*.json')):
-                x = etree.SubElement(policies,os.path.basename(fn).replace('_','').replace('.json',''))
+                try:
+                    x = etree.SubElement(policies,os.path.basename(fn).replace('_','').replace('.json',''))
```

Comment:
Happy to admit I'm not super sure what is happening here, but won't this still load `myrandomjson.json` and such? Shouldn't we just have it do something like `if fn in ('grading_policy.json', 'policy.json'):` in the loop or 
```python
policy_files = [fn for fn in os.listdir(pdir) if re.match(r'(grading_)?policy.json', fn)]
for fn in policy_files:
...
```
? Also, probably out of scope, but showing in the diff, why not `with open(fn) as file:` so we don't leave orphaned file handlers?